src/main/resources/sample_config.properties

```diff
@@ -0,0 +1,8 @@
+### WARNING: DO NOT EDIT THIS FILE DIRECTLY!!
+### Create a copy of this file and customize it
+
+### Sample config.properties file
+user=root
+password=root
+host=localhost
+port=3306
```

Comment:
This probably should not be here. These types of config really should _not_ go into config, they should be in auth.properties, which should not be in github.

Reply:
And auth.properties should be in .gitignore.

Reply:
Okay - I'll look into reworking the config files and adding auth.properties to .gitignore.